tests/test_data/test_datasets/test_openimages_dataset.py

```diff
@@ -14,7 +14,7 @@ def _create_ids_error_oid_csv(
     fake_csv_file,
 ):
     label_description = ['/m/000002', 'Football']
-    with open(label_file, 'w') as f:
+    with open(label_file, 'w', newline='') as f:
```

Comment:
# `newline=''` is used to avoid index error of out of bounds
    # in Windows system